provisioning/miniprov/hare_mp/templates/hare.config.conf.tmpl.3-node

```diff
@@ -42,11 +42,20 @@
         "cvg": [
           {
             "data_devices": [
-              "TMPL_DATA_DEVICE_1",
-              "TMPL_DATA_DEVICE_2"
+              "TMPL_DATA_DEVICE_11",
```

Comment:
If the first digit in the suffix denotes the node number I think it would be good to add a comment stating the convention  as `TMPL_DATA_DEVICE_NODE#DEV#`

Reply:
@mssawant  I have added comment in https://github.com/Seagate/cortx-hare/wiki/Hare-provisioning-for-3-node-deployment
as we can not add comment in above file

Reply:
Okay.